devicon.json

```diff
@@ -2468,8 +2716,30 @@
         "color": "#5E5185",
         "aliases": []
     },
+    {
+        "name": "handlebars",
+        "altnames": [
+            "handlebarsjs"
+        ],
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#000000",
+        "aliases": []
+    },
```

Comment:
Duplicate entry of `handlebars`

Reply:
6e9a895871f9c489551e347f4b9c05ee0830f8ae